CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 0.14.0 (2022-01-13)
+
+- Add an account level S3 public access policy to by default block public access to all S3 buckets within the landing zone accounts. ([#125](https://github.com/schubergphilis/terraform-aws-mcaf-landing-zone/pull/125))
```

Comment:
Tried to make it read better and specified it's for the "core" accounts and not other accounts created via the service catalogue or enrolled into the landing zone.

```suggestion
- Add an account level S3 public access policy to block public access to all S3 buckets by default within the landing zone core accounts. ([#125](https://github.com/schubergphilis/terraform-aws-mcaf-landing-zone/pull/125))
```